go.mod

```diff
@@ -114,6 +113,7 @@ require (
 	github.com/tyler-smith/go-bip39 v1.0.2 // indirect
 	go.uber.org/atomic v1.9.0 // indirect
 	go.uber.org/multierr v1.8.0 // indirect
+	go.uber.org/zap v1.21.0 // indirect
```

Comment:
why we continue needing zap here?

Reply:
It says `indirect`, maybe because the avalanchego logger uses zap?

Reply:
https://golangbyexample.com/direct-indirect-dependency-module-go/